proposals/0000-accessibilityapis-lists.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Accessibility States and Roles for lists
+author:
+- Harini Kannan
+date: 06/12/2019
+---
+
+# RFC0000: Accessibility States and Roles for lists
+
+## Summary
+
+This proposal adds new accessibility roles for supporting lists as well as new APIs for describing size of lists,nth item in the list etc.,
+
+This addresses Issues [2578](https://github.com/microsoft/react-native-windows/issues/2578) and [2579](https://github.com/microsoft/react-native-windows/issues/2579) from react-native-windows. An implementation of the same can be tracked through [this PR](https://github.com/microsoft/react-native-windows/pull/2743). 
+
+## Basic example
+
+React Native developers can use these new APIs to describe lists, listitems, size of lists, nth item in the list etc., to assistive technologies using simple code as shown below:
+
+```
+<FlatList
+  data={this.state.listdata}
+  accessibilityRole="list"
+  renderItem={({item}) =>
+     <View accessibilityRole="listItem" 
+           accessibilitySetSize ={this.state.listdata.count}
+           accessibilityPostInSet={item.id} >
+        <Text>{item.name}</Text>
+     </View>
+  }
+</FlatList>
+```
+
+
+## Motivation
+
+It is currently not possible for react-native developers to describe size of lists, nth item in the list etc., With these new APIs, react-native developers can provide more information on lists to assistive technologies which can then be more descriptive of list experiences. 
+
+## Detailed design
+
+### P1 APIs:
+The following roles will be added to [accessibilityRole](https://facebook.github.io/react-native/docs/accessibility#accessibilityrole-ios-android) APIs:
+- **list** Used when element should be treated as a list
+- **listItem** Used when element should be treated as a list item
```

Comment:
Can we make this `listitem` so it matches with the ARIA value. In general I'd like to see the role values move towards aligning with ARIA.

Reply:
Thanks, that sounds right. Updated.